src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,91 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.lang.Integer.parseInt;
+
+public class StringAddCalculator {
+    public static final String REGEX = ",|:";
+    public static final String CUSTOM_REGEX = "//(.)\n(.*)";
+    public static final int DELIMITER_GROUP = 1;
+    public static final int TEXT_GROUP = 2;
+
+    private static final Pattern CUSTOM_PATTERN = Pattern.compile(CUSTOM_REGEX);
+
+    public static int splitAndSum(String text) {
+        if (text == null || text.isEmpty()) {
+            return 0;
+        }
+
+        return sum(toInts(split(text)));
+    }
+
+    public static int sum(List<Integer> numbers) {
```

Comment:
```suggestion
    private static int sum(List<Integer> numbers) {
```
상수와 마찬가지로 내부 메서드는 private으로 막는 것이 좋겠네요 😄